injector_test.go

```diff
@@ -85,22 +84,31 @@ func TestProcess(t *testing.T) {
 
 			t.Run("single mark", func(t *testing.T) {
 				got, err := Process(context.Background(), []byte(tt.input), params)
-				require.NoError(t, err)
-				assert.JSONEq(t, tt.want, string(got))
+				if err != nil {
+					t.Error(err)
```

Comment:
слушай, извини, утро вечера мудренее, пару `t.Error` + `return` можно заменить на `t.Fatal`
```suggestion
					t.Fatal(err)
```

Reply:
Отлично!

Reply:
Сделал